submarine-cloud-v2/README.md

```diff
@@ -57,21 +64,25 @@ go build -o submarine-operator
 kubectl create ns submarine-admin
 kubectl apply -n submarine-admin -f artifacts/examples/example-submarine.yaml
 
-# Step3: "submarine-operator" will perform port-forwarding automatically.
+# Step3: Exposing Service
+# Using minikube ip + NodePort
+$ minikube ip  # you'll get the IP address of minikube, ex: 192.168.49.2
 
-# Step4: View workbench (127.0.0.1:32080) with your web browser
+# Step4: View workbench
+# http://{minikube ip}:32080, ex: http://192.168.49.2:32080
```

Comment:
In my experience, `minikube ip` does not work in the development environment (MacOS), and thus it would be better to add method2 (port-forward command).

Reply:
Oh I will add it

Reply:
In the following figure, the branch is at commit "SUBMARINE-841. Modify hard-coded URLs in E2E test to soft-coded URLs".
<img width="581" alt="截圖 2021-06-06 下午10 44 34" src="https://user-images.githubusercontent.com/20109646/120928880-1d066400-c719-11eb-8bda-d2f29af4d2b4.png">

Reply:
It seems that you miss the url to expose, but I don't know what's the reason.
![](https://user-images.githubusercontent.com/55401762/120966507-e6702e00-c798-11eb-9a7a-54b52d48041e.png)
I will add method2 to make sure everyone can access to the service.